src/client.ts

```diff
@@ -6107,17 +6107,17 @@ export class MatrixClient extends EventEmitter {
     public register(
         username: string,
         password: string,
-        sessionId: string,
-        auth: any,
-        bindThreepids: any,
-        guestAccessToken: string,
-        inhibitLogin: boolean,
+        sessionId: string | null,
```

Comment:
we don't generally describe `null`, but this is fine

Reply:
@turt2live It makes sense to add nulls when possible, as this will make it easier for us to turn on strictNullChecks in the future